rusqlite_migration/src/lib.rs

```diff
@@ -745,3 +799,11 @@ fn validate_foreign_keys(conn: &Connection) -> Result<()> {
         None => Ok(()),
     })
 }
+
+impl<'u> FromIterator<M<'u>> for Migrations<'u> {
```

Comment:
You are right, I should have implemented the trait instead of adding a `new_iter` method.